requirements.txt

```diff
@@ -4,3 +4,6 @@ python-dateutil==2.1
 six==1.12.0
 sleekxmpp==1.1.11
 freezegun==0.3.15
+
+requests~=2.24.0
+setuptools~=47.2.0
```

Comment:
Why?

Reply:
That was added automatically by IDE. I'll remove it.